docs/deployment/frameworks/helm.md

```diff
@@ -17,18 +17,45 @@ Before you begin, ensure that you have the following:
 
 ## Installing the chart
 
+First of all, ensure you have cloned the repo and navigate to the Helm Chart directory:
+
+```bash
+git clone https://github.com/vllm-project/vllm.git
+cd vllm/examples/online_serving/chart-helm
+```
+
 To install the chart with the release name `test-vllm`:
 
 ```bash
 helm upgrade --install --create-namespace \
   --namespace=ns-vllm test-vllm . \
-  -f values.yaml \
+  -f values.yaml \  # If you have another values file to override the default one, add another '-f values.override-yaml' line.
   --set secrets.s3endpoint=$ACCESS_POINT \
   --set secrets.s3bucketname=$BUCKET \
   --set secrets.s3accesskeyid=$ACCESS_KEY \
   --set secrets.s3accesskey=$SECRET_KEY
 ```
 
+If you have a AMD Radeon, you might as well add the `values.override.yaml` file with a content like this:
```

Comment:
can you put them into a file like `values.amd-override.yaml` in the same directory as `values.yaml` and give the full command for using the override here?